src/transformStateWithClones.js

```diff
@@ -1,13 +1,60 @@
 'use strict';
 
 /**
- * @param {Object} state
+ * @param {Object} localState
  * @param {Object[]} actions
  *
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const listOfClones = [];
+  const stateCopy = {
+    ...state,
+  };
+
+  for (const action of actions) {
+    const actionType = action['type'];
```

Comment:
If you know the exact key name, you should use the dot notion
```suggestion
    const actionType = action.type;
```